src/main/java/tfifteenfour/clipboard/logic/commands/editcommand/EditSessionCommand.java

```diff
@@ -21,8 +21,9 @@ public class EditSessionCommand extends EditCommand {
     public static final String COMMAND_TYPE_WORD = "session";
     public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
Should we also add `COMMAND_TYPE_WORD` here?
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD  + " " + COMMAND_TYPE_WORD
```